.github/workflows/main.yml

```diff
@@ -112,12 +112,16 @@ jobs:
       with:
         tinygo-version: 0.31.0
 
+    - uses: olafurpg/setup-scala@v11
```

Comment:
`olafurpg/setup-scala` is deprecated. 


https://github.com/sbt/setup-sbt

Reply:
Ah thanks. I've seen mentioning setup-scala is deprecated _because sbt is now included in the runners_ but turned out that's not the case anymore, that's why I reverted to it (without finding the new action to use).